### PR TITLE
[specific ci=Group1-Docker-Commands] Implement --hostname flag

### PR DESCRIPTION
**Pull request by caglar10ur**
Helps #5833

### CODE REVIEW

File: tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot [hunk 1]
```diff
@@ -147,3 +147,14 @@ Docker run and auto remove
     Should Be Equal As Integers  ${rc}  0
     ${output}=  Split To Lines  ${output}
     Length Should Be  ${output}  ${count}
+
+Docker run --hostname to set hostname and domainname
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --hostname vic.test hostname
```
Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot]: are you missing the image here?

Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot]: ... run --hostname vic.test ${busybox} hostname

**Comment by mhagen-vmware** [comment]
fixes https://github.com/vmware/vic/issues/4195

File: tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot (hunk 1 above)
Comment (caglar10ur) [inline tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot]: Yep, I'm missing it. Fixing now

**Pull request merged**